maven/WFLY-14934_Server_Plugin.adoc

```diff
@@ -132,18 +132,24 @@ WildFly. This change impacts ```run``` and ```start```goals).
 These configuration items are common to the ```provision``` and ```package``` goals. They are used to configure 
 Galleon to provision a server.
 
-* feature-packs: A list of feature-pack to install.
+* feature-packs: A list of feature-pack to install. This can be configured with the system property ``wildfly.provisioning.feature-packs`` containing 
+a comma separated list of feature-pack locations in the form of Maven coordinates GroupId:ArtifactId:Version or producer reference.
 * log-provisioning-time: Whether to log provisioning time at the end.
 * offline-provisioning: Whether to use offline mode when the plugin resolves an artifact.
 * galleon-options: Arbitrary Galleon options used when provisioning the server.
-* provisioning-dir: The name of the directory relative to the Maven project build directory used when the server is provisioned. 
+* provisioning-dir: The path to the directory where to provision the server. Can be an absolute path or a path relative to the Maven project build directory. 
+When absolute and outside the project build directory, the provisioned server will be not deleted when clean occurs.
 It defaults to ``server`` and will result in the server being provisioned in ``${project.build.directory}/server`` (which is resolved to ``target/server`` by default)
 * provisioning-file: The path to the provisioning.xml file to use, by default ```<project base dir>/galleon/provisioning.xml``` is used. If some feature-packs have been set, 
 the provisioning file is being ignored.
 * record-provisioning-state: Whether to record provisioned state in .galleon directory. When false, the file ```.wildfly-maven-plugin-provisioning.xml``` is generated in the server home directory.
 It contains Galleon provisioning information used to provision this server.
-* layers: A set of layers to include when building-up a custom configuration.
-* excluded-layers: A set of layers to exclude when building-up a custom configuration.
+* layers: A set of layers to include when building-up a custom configuration. This can be configured with the system property ``wildfly.provisioning.layers`` containing a comma separated list of layers.
+* excluded-layers: A set of layers to exclude when building-up a custom configuration.  This can be configured with the system property ``wildfly.provisioning.layers.excluded`` containing a comma separated list of layers.
+* layers-configuration-file-name: The name of the configuration file generated from layers. Default value is ``standalone.xml``. 
+If no ``layers`` have been configured, setting this parameter is invalid.
+* overwrite-provisioned-server: By default if the server referenced from the ``provisioning-dir`` exists when provision/package goals are called, the provisioning/packaging is disabled. By setting
+this parameter to false, the provisioned server will get deleted and provisioning/packaging will occur.
```

Comment:
> By setting this parameter to false, the provisioned server will get deleted and provisioning/packaging will occur.

If false is default behaviour, then this has to be set to `true`. Or not?

Reply:
This was a typo, I fixed it to be true.